main.gms

```diff
@@ -2,8 +2,74 @@
 *' 
 *' @description This is the OPEN-PROM model, the open version of the world energy model PROMETHEUS of E3-Modelling.
 *' @code
-*' *** Preliminaries 
-*' 
+*' *** preliminaries 
+*' ----
+*' Coding Etiquette:
+*' ----
+*' Below, you'll find the coding etiquette, a collection of conventions designed to streamline the development process and enhance code readability.
+*' Please uphold these standards as you code!
+*' ----
+*' [Q/V/i][Cost/Cap/Dem/Cons/Price/etc.][more specific, if needed][Elec/Tr/Ind/Dom][etc.]
```

Comment:
An explanation of what this is would help here, something along the lines of "The names of the objects in the code are defined as follows:"

Reply:
Also, better to formulate it like this:
[ prefix ][ object type (e.g. Capacity, Cost, Share, etc.) ][ more specific information, if needed ][ object scope, e.g. Transport, Electricity, etc. ][ more specific information, if needed ]
Example:
[ Q/V/i ][ Cost/Cap/Dem/Cons/Price/etc. ][ Total ][ Elec/Tr/Ind/Dom ][ etc. ]